apps/pie-microsite/src/assets/styles/_variables.scss

```diff
@@ -1 +1,2 @@
-$content-max-width: 80ch;
+$content-max-width: 65ch;
```

Comment:
we could probably rename this too to make sure it's clearly referring to typography (if it is)